post-scan-actions/azure-python-promote-or-quarantine/template.json

```diff
@@ -56,21 +70,71 @@
             "metadata": {
                 "description": "The method by which files were quarantined.<br/>(Options: move, copy)."
             }
+        },
+        "VNETResourceID": {
+            "defaultValue": "",
+            "type": "String",
+            "metadata": {
+                "description": "<b>[VNet Only]</b><br />This field requires the resource ID of the existing virtual network (VNet) that the deployed resource will be associated with. If you wish to deploy the plugin into a VNet, please see <a href='https://github.com/trend-ryan-c-wu/cloudone-filestorage-plugins/tree/master/post-scan-actions/azure-python-promote-or-quarantine#2-deploy-function-app-with-vnet-integration'>Deploy Function App with VNet Integration</a> for detailed instructions and ensure that the required resources have been created.<br /> If a VNet is not required, please leave this field blank."
```

Comment:
Remind to update the URL to this repo

Reply:
Please update the github URL to this repo's.

Reply:
Sorry, fixed.